ui/next/config.js

```diff
@@ -137,6 +139,9 @@ System.config({
     "github:jspm/nodelibs-zlib@0.1.0": {
       "browserify-zlib": "npm:browserify-zlib@0.1.4"
     },
+    "npm:ByteBuffer@0.2.6": {
```

Comment:
It's used by the uiData reducer which is used by helpus.tsx

On Fri, Jun 3, 2016, 4:28 PM Tamir Duberstein <notifications@github.com>
wrote:

> In ui/next/config.js
> <https://github.com/cockroachdb/cockroach/pull/7020#discussion_r65770518>:
>
> > @@ -137,6 +139,9 @@ System.config({
> >      "github:jspm/nodelibs-zlib@0.1.0": {
> >        "browserify-zlib": "npm:browserify-zlib@0.1.4"
> >      },
> > +    "npm:ByteBuffer@0.2.6": {
>
> also, where is this used in the browser?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/7020/files/8e3390031bdfa925fdf72c8ca14ca730f994baa8#r65770518>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADhVbstNOH28nJvusC-lKPVoDlVSJnp_ks5qII5qgaJpZM4ItAS->
> .
>